pkgs/flox-cli/default.nix

```diff
@@ -39,8 +39,10 @@ let
       NIX_BIN = "${nix}/bin/nix"; # only used for nix invocations in tests
       GIT_PKG = gitMinimal;
 
+      # Our own tools
       WATCHDOG_BIN =
         if flox-watchdog == null then "flox-watchdog" else "${flox-watchdog}/bin/flox-watchdog";
+      FLOX_INTERPRETER = flox-activation-scripts;
```

Comment:
suggestion blocking: also pass this to `flox-cli-tests` otherwise we'll have to re-enter the dev shell to run integration tests

Reply:
I should have marked this one as `suggestion followup` as well

Reply:
Fixed